### PR TITLE
Create views for ResolutionEmail model

### PR DESCRIPTION
**Pull request by wagnersthub**

### CODE REVIEW

File: resotool/resoapp/templates/resoapp/resolutionemail_detail.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+{% block content %}
+
+<h2> {{ resolutionemail.resolution.title }}</h2>
+<table class="table email_table">
+	<tr>
+		<th>Adressierte Person</th><td> {{ resolutionemail.recipient.name}}</td>
+	</tr>
+	<tr>
+		<th>E-mail Adresse</th><td> {{ resolutionemail.recipient.postal_address}}</td>
```
Comment (behrmann) [inline resotool/resoapp/templates/resoapp/resolutionemail_detail.html]: Shouldn't this be `resolutionemail.recipient.email`?

**Review by behrmann — CHANGES_REQUESTED** [review]

**Comment by behrmann** [comment]
Haven't tested this yet, but I had a first cursory view; will have a closer look tomorrow.

I recommend squashing this in the end.

File: resotool/resoapp/static/resoapp/style.css [hunk 2]
```diff
@@ -1 +1,14 @@
 @import "bootstrap/css/bootstrap.css";
+
+@media (min-width: 601px) {
```
Comment (behrmann) [inline resotool/resoapp/static/resoapp/style.css:3]: The 601 pixels as `min-width` is oddly specific. Where do they come from and why is this `min-width` smaller than the `max-width` below? Sorry if this is obvious, when it comes to CSS I don't know anything.

Comment (scattenlaeufer) [reply resotool/resoapp/static/resoapp/style.css:3]: With my understanding of CSS, the current settings in `style.css` should be correct. Those settings state that for browsers with a width of 601px or more should use the first block, giving the email table a fixed width. Browsers with a width of 600px or below should use the second, making the email table use the entire width of the given area.

The oddly specific settings are just for now. They probably need to be adjusted once we decided on how(/whether) we want to design the site as a whole.

File: resotool/create_dummy_resos.py [hunk 3]
```diff
@@ -112,6 +112,47 @@
 reso1.save()
 reso1.user_set.add(user1, user2, user3)
 
+recipient1 = Recipient(name='AFD', opening='Hallo,', postal_address='afdhausen')
```
Comment (scattenlaeufer) [inline resotool/create_dummy_resos.py]: In regard of @behrmann's comment on `resolutionemail_detail.html`, all the recipients here should also get an email address to be shown in the table.

**Review by scattenlaeufer — CHANGES_REQUESTED** [review]
I tested this PR on my machine and everything seems to work fine, so just make those two changes and this is a go from me.

In addition we should change the way we display the meta data of a resolution in the same kind of table we do it here for emails. I'll open an issue to change that.

**Review by scattenlaeufer — APPROVED** [review]

**Review by behrmann — APPROVED** [review]

**Pull request merged**